MultiPoolMiner.ps1

```diff
@@ -75,31 +78,25 @@ param(
     [Parameter(Mandatory = $false)]
     [Switch]$UseFastestMinerPerAlgoOnly = $false, #Use only use fastest miner per algo and device index. E.g. if there are 2 miners available to mine the same algo, only the faster of the two will ever be used, the slower ones will also be hidden in the summary screen
     [Parameter(Mandatory = $false)]
-    [Switch]$IgnoreFees = $false, #if $true MPM will ignore miner and pool fees for its calculations (as older versions did)
-    [Parameter(Mandatory = $false)]
-    [Switch]$ShowPoolBalances = $false,
-    [Parameter(Mandatory = $false)]
-    [Switch]$ShowPoolBalancesExcludedPools = $false,
-    [Parameter(Mandatory = $false)]
-    [Switch]$ShowPoolBalancesDetails = $false,
+    [Alias("IgnoreFees")]
```

Comment:
This should be left as 'IgnoreFees'

Version 3.3.0 will include power cost calculations and will introduce parameter 'IgnorePowerCosts'

Miner fees and power costs are not the same.